go/consensus/tendermint/apps/governance/governance.go

```diff
@@ -71,12 +71,14 @@ func (app *governanceApplication) ExecuteTx(ctx *api.Context, tx *transaction.Tr
 		if err := cbor.Unmarshal(tx.Body, &proposalContent); err != nil {
 			return err
 		}
+		ctx.SetPriority(AppPriority)
```

Comment:
Could just have a single set above given that this is the same for both cases? In case the transaction is later found to be invalid it will be discarded from the mempool anyway due to failing checks.

Also in other services/apps.